server/graphql/invoice_line/src/mutations/inbound_shipment_line/line/insert.rs

```diff
@@ -26,8 +27,8 @@ pub struct InsertInput {
     pub sell_price_per_pack: f64,
     pub expiry_date: Option<NaiveDate>,
     pub number_of_packs: u32,
-    pub total_before_tax: f64,
-    pub tax: Option<TaxUpdate>,
+    pub total_before_tax: Option<PriceInput>,
```

Comment:
the struct has a `total_before_tax` property, so we'll end up with `total_before_tax.total_before_tax` won't we? 😕 
Why not just a float? or have a `pub price` property here, with all the tax bits in it?